build-system/pr-check/validator-tests.js

```diff
@@ -49,6 +50,10 @@ function prBuildWorkflow() {
     timedExecOrDie('amp validator');
   }
 
+  if (buildTargetsInclude(Targets.VALIDATOR)) {
```

Comment:
The cpp tests now only runs when the PR changes validator files

Reply:
Excellent! Can we move this to the bottom, so that the faster running `amp validator` and `amp validator-webui` can fail-fast, and we run the CPP tests only after both of them pass?

Reply:
Done